packages/eas-cli/src/update/utils.ts

```diff
@@ -19,40 +21,46 @@ export type UpdateGroupDescription = FormatUpdateParameter & {
   runtimeVersion: string;
 };
 
+export type FormattedUpdateGroupDescription = {
+  message: string;
+  group: string;
+  platforms: string;
+  runtimeVersion: string;
+};
+
+export type FormattedUpdateGroupDescriptionWithBranch = FormattedUpdateGroupDescription & {
+  branch: string;
+};
+
 export const UPDATE_COLUMNS = [
   'Update message',
   'Update runtime version',
   'Update group ID',
   'Update platforms',
 ];
 
-export function getUpdateGroupsWithPlatforms<
-  UpdateFragment extends Pick<Update, 'platform' | 'group'>
->(updates: UpdateFragment[]): (UpdateFragment & { platforms: string })[] {
-  return Object.values(groupBy(updates, updates => updates.group)).map(updateGroup => ({
-    ...updateGroup[0],
-    platforms: formatPlatformForUpdateGroup(updateGroup),
-  }));
-}
+export const UPDATE_COLUMNS_WITH_BRANCH = ['Branch', ...UPDATE_COLUMNS];
 
 export function getPlatformsForGroup({
-  group,
-  updates,
+  group = '',
+  updates = [],
```

Comment:
Can you explain the defaults? They look odd and unnecessary.

Reply:
sure. What I'm thinking here is that functionally we want to have `getPlatformsForGroup` return `N/A` when handling null/undefined/invalid input. It also seems to make sense to have `formatPlatformForUpdateGroup` always be what returns the string.

So with that in mind, we'll either need to default values here or default them in the function body. `groupBy` cannot operate on `undefined`, it needs some kind of array. And we cannot index a record on `undefined`, we need some kind of string.

This seemed to me the simplest option but happy to change it.

Reply:
Sorry for not being clear. I mostly meant `group = ''`. This particular default looks strange.
Why? Let's say you have passed `group = undefined`. As a result, line 52 will try to access `groupedUpdates['']`. It's obviously `undefined`. `formatPlatformForUpdateGroup` support `undefined` but in my opinion, this is quite a weird flow. I mean you're treating `''` as a falsy value here. It's better to use more meaningful falsy values like `null`  or `undefined`, and them properly.

Reply:
Also see the comment below.